package.json

```diff
@@ -17,13 +19,23 @@
     "istanbul": "^0.3.13",
     "jshint": "^2.5.8",
     "jshint-stylish": "^2.0.1",
+    "karma": "^0.13.14",
+    "karma-chai": "^0.1.0",
+    "karma-coverage": "^0.5.3",
+    "karma-mocha": "^0.2.0",
+    "karma-mocha-reporter": "^1.1.1",
+    "karma-phantomjs-launcher": "^0.2.1",
+    "karma-sinon": "^1.0.4",
+    "mocha": "^2.3.3",
+    "phantomjs": "^1.9.18",
     "plato": "^1.4.0",
```

Comment:
@AurelioDeRosa I thought I removed that. Not at the moment, but it might be good to add later. Agreed with adding Gulpfile to linter, will update commit with both fixes.